### DISCUSSION TITLE
how to add surround to select string

### DISCUSSION THREAD

**Discussion by WingDust** [kylechui/nvim-surround#165] [Q&A] [answered]
`V.cc(this.selected)` 
how to turn `(V.cc(this.selected))`

**Comment by cbochs** [comment, marked as answer]
```
Before:
     V.cc(this.selected)
     ^^^^ place cursor outside the inner set of parentheses

Motion: ysiW)

After:
    (V.cc(this.selected))
```

**Reply by kylechui** [reply]
Two things: You should be able to have your cursor anywhere in the text (including in the first set of parentheses) when you do `ysiW)`, and an alternative (assuming this is the only text on the line) is `yss)`
